src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -50,7 +50,7 @@ type TaskSeq =
     /// <summary>
     /// Returns the length of the sequence of all items for which the <paramref name="predicate" /> returns true.
     /// This operation requires the whole sequence to be evaluated and should not be used on potentially infinite sequences.
-    /// If <paramref name="predicate" /> is asynchronous, use <see cref="TaskSeq.lengthByAsync" />.
+    /// If <paramref name="predicate" /> is asynchronous, consider using <see cref="TaskSeq.lengthByAsync" />.
```

Comment:
```suggestion
    /// If <paramref name="predicate" /> is asynchronous, see <see cref="TaskSeq.lengthByAsync" />.
```

Reply:
eh, I actually took the "consider using" from an earlier PR by you that fixed that on some other function descriptions. 

Well, new insights, you're the language warrior 😆. We can always revert back. Unf. I already published this change prior to seeing your comments.

Reply:
not the first time I've criticised my work knowingly and/or unknowingly. It's a sort of red/green/refactor cycle where I add words and then take them away and/or replace with shorter or mode direct. IOW simplify :D